presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -133,11 +134,11 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
                 .orElseGet(() -> listTables(session, prefix.getSchema()));
         for (SchemaTableName tableName : tables) {
             try {
-                JdbcTableHandle tableHandle = jdbcClient.getTableHandle(tableName);
-                if (tableHandle == null) {
+                Optional<JdbcTableHandle> tableHandle = jdbcClient.getTableHandle(tableName);
```

Comment:
Let's invert this (the existing behavior of skipping a single line using `continue` seems silly):
```java
jdbcClient.getTableHandle(tableName).ifPresent(tableHandle ->
        columns.put(tableName, getTableMetadata(session, tableHandle).getColumns()));